ast/misc.go

```diff
@@ -84,7 +84,8 @@ type AuthOption struct {
 type ExplainStmt struct {
 	stmtNode
 
-	Stmt StmtNode
+	Stmt   StmtNode
+	Format string
```

Comment:
Could we use "const" instead of a string?

Reply:
no we can't, when a input explain format is not supported, we should return an error like
`ERROR 1791 (HY000): Unknown EXPLAIN format name: '%s'`